package.json

```diff
@@ -59,6 +58,22 @@
     "plugin:build:commit": "lerna run build:commit --ignore=\"@grafana/*\" --ignore=\"@grafana-plugins/input-datasource\"",
     "plugin:build:dev": "lerna run dev --ignore=\"@grafana/*\" --ignore=\"@grafana-plugins/input-datasource\""
   },
+  "nx": {
+    "targets": {
+      "start": {
+        "dependsOn": [
+          "grafana:themes:generate",
+          "@grafana-plugins/grafana-testdata-datasource:build",
+          "@grafana-plugins/grafana-pyroscope-datasource:build",
+          "@grafana-plugins/grafana-azure-monitor-datasource:build",
+          "@grafana-plugins/grafana-cloud-monitoring-datasource:build",
+          "@grafana-plugins/parca:build",
+          "@grafana-plugins/tempo:build"
+        ]
+      }
+    },
+    "parallel": 5
```

Comment:
this can't live in the nx.json file? Weird that we have/need both.

Reply:
I had some weirdness with running top level scripts with NX and this seemed like the more reliable place for it to work. I'll try fiddling some more as I'd also like it all in a single place. 👍

Reply:
Because of the way top-level scripts work with NX it doesn't seem we can put this info in the nx.json file. I've moved it over to a project.json file and cleaned up the repetition.